lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -668,7 +659,8 @@ def write_request_via_grpc(entries:,
                   end,
         labels: labels.map do |k, v|
           [k.encode('utf-8'), convert_to_utf8(v)]
-        end.to_h
+        end.to_h,
+        partial_success: @partial_success
```

Comment:
The comment in line 650 is no longer correct.

Reply:
Ah right. Removed.